### PR TITLE
Update with changes

### PR DESCRIPTION
**Pull request by misteryeo**
- Update from Slack to Discourse forum
- Remove mentions of Singer including broken examples

### CODE REVIEW

**Review by marcosmarxm — APPROVED** [review]
awesome

File: docs/connector-development/cdk-python/README.md [hunk 1]
```diff
@@ -1,16 +1,19 @@
 # Connector Development Kit (Python)
 
+    :::info
+    Singer Taps are no longer supported and contributions of this type will not be accepted by the Airbyte team.
```
Comment (sherifnada) [inline docs/connector-development/cdk-python/README.md:4]:
```suggestion
    The Airbyte team will no longer accept contributions of Singer-based connectors into the Airbyte repo. However, you can still build and use Singer-based connectors in Airbyte using the CDK.
```

File: docs/connector-development/cdk-python/README.md [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
+    :::
+
 The Airbyte Python CDK is a framework for rapidly developing production-grade Airbyte connectors. The CDK currently offers helpers specific for creating Airbyte source connectors for:
 
 * HTTP APIs \(REST APIs, GraphQL, etc..\)
-* Singer Taps
```
Comment (sherifnada) [inline docs/connector-development/cdk-python/README.md:6]: The CDK can still work with Singer. Are you saying this should be hidden/we shouldn't facilitate it for people?  Seems harmless to keep this here.

File: docs/connector-development/cdk-python/README.md (hunk 1 above)
Comment (sherifnada) [reply docs/connector-development/cdk-python/README.md:4]: I don't think this should be so high up in the docs -- we're giving Singer more of a central spot than it needs imo. Can it go in the bottom of this section?

File: docs/connector-development/cdk-python/README.md (hunk 2 above)
Comment (marcosmarxm) [reply docs/connector-development/cdk-python/README.md:6]: Maybe add a note:
* Singer Taps (you can build connectors based on Singer but Airbyte doesn't accept any new singer based connectors?)

**Comment by Amruta-Ranade** [comment]
Thanks for the PR, Andy! Can we delete the docs references to the github-singer source: https://github.com/airbytehq/airbyte/search?l=Markdown&q=source-github-singer

File: docs/connector-development/cdk-python/README.md (hunk 1 above)
Comment (sherifnada) [reply docs/connector-development/cdk-python/README.md:4]: > What is the concern around it being a 'central' spot?

it seemed "funny" that the very first thing we say about our own CDK in our own docs is a statement about Singer. We shouldn't be so competitor focused, but rather just focus on our own product.

I like Marcos' suggestion of embedding this note in-line with where we say the CDK supports singer taps

Comment (misteryeo) [reply docs/connector-development/cdk-python/README.md:4]: Cool I'll move forward with that!

**Review by sherifnada — APPROVED** [review]

**Review by Amruta-Ranade — APPROVED** [review]

**Pull request merged**